recipe/meta.yaml

```diff
@@ -11,21 +11,19 @@ package:
 build:
   number: {{ build_num }}
   string: {{ build_num }}_{{ python_abi_tag }}
-  track_features:  # [python_implementation == "pypy"]
-    - pypy         # [python_implementation == "pypy"]
+  track_features:         # [python_implementation != "cpython"]
+    - pypy                # [python_implementation == "pypy"]
+    - native_graalpy      # [python_implementation == "graalpy"]
 
 requirements:
-  run:
-    - python {{ python_impl_version }}.*                    # [python_implementation == "cpython"]
-    - pypy{{ python_version }} {{ python_impl_version }}.*  # [python_implementation == "pypy"]
   run_constrained:
-    - pypy <0a0                                                                         # [python_implementation == "cpython"]
-    - python {{ python_version }}.* *_{{ python_impl_version.replace('.', '') }}_pypy     # [python_implementation == "pypy"]
+    - python {{ python_version }}.* *_cpython                                                # [python_implementation == "cpython"]
+    - python {{ python_version }}.* *_{{ python_impl_version.replace('.', '') }}_pypy        # [python_implementation == "pypy"]
+    - python {{ python_version }}.* *_native_graalpy                                         # [python_implementation == "graalpy"]
```

Comment:
@timfel, is this okay?

Reply:
I would prefer a scheme similar to pypy, since we don't currently keep the abi stable for graalpy even across minor releases. So something like
```
python {{ python_version }}.* *_{{ native }}{{ python_impl_version.replace('.', '') }}_graalpy
```